frontend/src/pages/MessagingQueues/MQDetails/MQTables/MQTables.tsx

```diff
@@ -126,7 +131,13 @@ function MessagingQueuesTable({
 	tableApi: (
```

Comment:
why is `tableApi` getting passed on like this ?

Reply:
this is a common table component used by different pages having different payloads and APIs in them.